metronome/hotstuff/service/src/io/iohk/metronome/hotstuff/service/ConsensusService.scala

```diff
@@ -357,8 +387,7 @@ class ConsensusService[F[_]: Timer: Concurrent, N, A <: Agreement: Block](
         // should be offloaded to another queue.
         //
         // Save the Commit Quorum Certificate to the view state.
-        saveCommitQC(commitQC) >>
-          blockExecutionQueue.offer(effect)
+        blockExecutionQueue.offer(effect)
```

Comment:
The comment about saving the QC and the pattern match are now obsolete, right?

Reply:
True, thanks!